src/main/resources/messages/MessageBundle_ru.properties

```diff
@@ -68,3 +68,5 @@ settings.general.sticker.reset.margins=Сбросить отступы Стик
 settings.general.content.sticker.margin.info=Кликните два раза по стикерам, чтобы сохранить текущую позицию.
 settings.general.content.sticker.ignore.scaling=Игнорировать масштабирование
 settings.general.randomizer.install=Выбирать тему случайно
+settings.general.sticker.hide-on-hover=Hide on hover
```

Comment:
@Dragon-0609 I've added some more message bundle stuff. If you would please translate these to Russian, that'd be great.
If this PR is still up, please submit it to `cleanup` branch, thanks!